pkg/prm/status.go

```diff
@@ -0,0 +1,44 @@
+package prm
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/Masterminds/semver"
+)
+
+type Status struct {
+	PuppetVersion *semver.Version
+	Backend       BackendType
+	BackendStatus BackendStatus
+}
+
+func (p *Prm) GetStatus() (status Status) {
+	// Need to mock calls to prm.LoadConfig() so we can set RunningConfig?
+	status.PuppetVersion = RunningConfig.PuppetVersion
+	status.Backend = RunningConfig.Backend
+	status.BackendStatus = p.Backend.Status()
+
+	return status
+}
+
+func FormatStatus(status Status, outputType string) (statusMessage string, err error) {
+	if outputType == "json" {
+		jsonBytes, err := json.Marshal(status)
+		if err == nil {
+			statusMessage = string(jsonBytes)
+		}
+	} else {
+		messageLines := []string{}
```

Comment:
Minor comment, don't really mind, but would a `StringBuilder` be better here?

Reply:
Probably! I didn't know that existed 😂 😭